### PR TITLE
Added documentation, fixed some tests, some cleanup.

### PR DESCRIPTION
**Pull request by SeleniumK**

### CODE REVIEW

**Comment by SeleniumK** [comment]
Opened a pull request for the directed unweighted graph. Do not merge in -- master currently has weighted graph in place. PR just for canvas resubmission.

File: src/simple_graph.py [hunk 1]
```diff
@@ -33,21 +39,24 @@ def add_edge(self, n1, n2):
...
+        return n in self.dict.keys()
 
     def neighbors(self, n):
+        """Return all neighbors of a node in graph."""
         return self.dict[n]
 
     def adjacent(self, n1, n2):
-        return True if n2 in self.dict[n1] else False
+        """Determine if edge exists in graph. Return Boolean value."""
+        return n2 in self.dict[n1]
```
Comment (nhuntwalker) [inline src/simple_graph.py:62]: This still needs to throw an error if n2 is not a node in the graph.

File: tests/test_simple_graph.py [hunk 2]
```diff
@@ -1,71 +1,74 @@
...
 
+
 def test_sgraph_add_edge_added_key():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_node(2) 
-    test_graph.add_edge(2,3)
+    test_graph.add_node(2)
+    test_graph.add_edge(2, 3)
     assert 3 in test_graph.nodes()
```
Comment (nhuntwalker) [inline tests/test_simple_graph.py:55]: You haven't checked if the edge was actually added, just that the node was added.

File: tests/test_simple_graph.py [hunk 3]
```diff
@@ -1,71 +1,74 @@
...
 
+
 def test_sgraph_add_edge_no_nodes():
     from src.simple_graph import Graph
     test_graph = Graph()
-    test_graph.add_edge(2,3)
+    test_graph.add_edge(2, 3)
     assert 3 in test_graph.nodes() and 2 in test_graph.nodes()
```
Comment (nhuntwalker) [inline tests/test_simple_graph.py:62]: Again you haven't checked if the edge was added, just that the nodes were added.

**Pull request closed without merging**